src/executables/csql.c

```diff
@@ -1751,6 +1760,7 @@ csql_execute_statements (const CSQL_ARGUMENT * csql_arg, int type, const void *s
   DB_QUERY_TYPE *attr_spec = NULL;	/* result attribute spec. */
   int total;			/* number of statements to execute */
   bool do_abort_transaction = false;	/* flag for transaction abort */
+  PT_NODE *statements;
```

Comment:
If it is for single statement parse tree, use **statement**

Reply:
Thank you, i will change code as your recommend.